Dockerfile

```diff
@@ -0,0 +1,43 @@
+# Build and test stage
+FROM python:3.8-buster as build
+
+# Upgrade pip
+RUN pip install -U pip
+
+WORKDIR /app
+
+COPY . /app
+
+#install dependencies
+RUN pip install -r requirements.txt
+
+#run unit tests
+RUN python3 -m unittest tests.arcfeed_cms_filtertest
+
+#Check styling
+RUN flake8 app.py
+
+#create deployment.zip
+RUN chalice package .
+
+# Uploading stage
+FROM python:3.8-buster as upload
+
+COPY --from=build /app /app
+
+WORKDIR /app
+
+ENV AWS_DEFAULT_REGION="us-east-1" \
+  BUCKET="pd-sophi-artifacts" \
+  SUBDIRECTORY="platform/warehousing/serverless" \
+  FILE_NAME="SOPHI4-616-ArcCms_kinesis_feed_filter_.zip" \
+  PACKAGE_NAME="deployment.zip" \
+  AUTO_DEPLOY="false"
```

Comment:
we should change the default values here to be general purpose, keep in mind that this is a public repository and it won't have access to things like platform and that zip file probably doesn't make sense in that context unless you've an examples folder and reference that.

Reply:
ok .. let me make then env variable that can be passed will building the docker image them..